client/src/components/MultiJurisdictionAudit/RoundManagement/_mocks.ts

```diff
@@ -0,0 +1,95 @@
+import { IAuditBoard } from '../useAuditBoards'
+
+export const auditBoardMocks: {
+  [key in
+    | 'empty'
+    | 'single'
+    | 'double'
+    | 'noBallots'
+    | 'started'
+    | 'signedOff']: IAuditBoard[]
+} = {
+  empty: [],
+  single: [
+    {
+      id: 'audit-board-1',
+      name: 'Audit Board #01',
+      signedOffAt: '',
```

Comment:
i believe this is `null` until the audit board signs off

Reply:
@jonahkagan The type is `string` so should I update that to `string | null`?

Reply:
oh yeah good call. guess i messed up that type originally